### PR TITLE
[17.0][FW] edi_oca: missing FW from 16.0

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
In the migration to V17 several commits were left behind in V16.

Depends on https://github.com/OCA/queue/pull/705

CC @etobella @simahawk

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
@simahawk @etobella @MiquelRForgeFlow ready for review :)

**Comment by simahawk** [comment]
@JordiMForgeFlow LG. Could you please update the blacklist of PRs?
If you are not using oca-port you can update the file manually, like https://github.com/OCA/edi/blob/14.0/.oca/oca-port/blacklist/edi_oca.json

**Comment by JordiMForgeFlow** [comment]
@simahawk what is expected to be included in this blacklist? The PRs that have been FW?

**Comment by simahawk** [comment]
My bad. Short answer is: no. Lemme rephrase: is there any PR that shouldn't be fwd ported or that has been partially fwd ported here (eg: leaving behind irrelevant commits)? If yes, we should blacklist them to not get bothered the next time we run oca-port to check what's needed to be ported.
Hope is clear now! :blush:

**Comment by JordiMForgeFlow** [comment]
@simahawk Then I don't think we need to add anything, as I have included everything that was missing from V16. I actually also included them in the V18 migration PR: https://github.com/OCA/edi-framework/pull/115

I can see you merged https://github.com/OCA/edi-framework/pull/93 a couple of days ago, I could add it here as well if it looks good to you :)

**Comment by simahawk** [comment]
@JordiMForgeFlow ok, beautiful!
#93 can be included, definitely. Thanks!

**Comment by JordiMForgeFlow** [comment]
@simahawk we will need to have https://github.com/OCA/edi/pull/1071 merged

File: edi_oca/views/edi_exchange_record_views.xml [hunk 1]
```diff
@@ -93,6 +93,23 @@
                     />
                 </header>
                 <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-calendar"
+                            name="action_view_related_queue_jobs"
+                            attrs="{'invisible': [('related_queue_jobs_count', '=', 0)]}"
```
Comment (SilvioC2C) [inline edi_oca/views/edi_exchange_record_views.xml:102]: This needs fixing, `attrs` and `states` are no longer supported since v17.0

**Comment by SilvioC2C** [comment]
@JordiMForgeFlow hey, since I was in a hurry, I opened [a new PR](https://github.com/OCA/edi-framework/pull/125) based on your PR, where I also rebased the branch against branch `17.0` and migrated a few of the forward-ported commits

**Comment by JordiMForgeFlow** [comment]
@SilvioC2C fine for me :)

**Pull request closed without merging**